.github/workflows/integration_tests.yml

```diff
@@ -422,23 +358,14 @@ jobs:
         shell: bash
         run: |
           pip install -r scripts/gha/requirements.txt
-      - name: Setup Unity
-        uses: kuler90/setup-unity@v1
+      - id: unity_setup
+        uses: ./gha/unity
         timeout-minutes: 30
         with:
-          unity-version: 2020.3.34f1
-          # Install ios for the editor tools
-          unity-modules: ios
```

Comment:
For playmode tests I had to install the iOS tools for Crashlytics and Messaging. Can you add this back in, and make sure that those tests pass.

Reply:
Is the iOS tool required only on Windows machine?

Reply:
It seems that the ios module only required on Windows machines.
Test result: https://github.com/firebase/firebase-unity-sdk/actions/runs/3473188998